test/handler/exampleauthfilter/exampleauthfilter.go

```diff
@@ -35,6 +38,9 @@ const (
 	commitFunc                    = "commit"
 	verifyProposalSignatureFunc   = "verifyProposalSignature"
 	validateProposalResponsesFunc = "validateProposalResponses"
+	putCASFunc                    = "putcas"
```

Comment:
Casing not consistent with other functions, nit (doesn't have to be fixed)

Reply:
I'll change it next time. This is just used for tests.

Reply:
Actually, it looks like the verifyProposalSignature and validateProposalResponses are not consistent with the rest of the function names.